activerecord/lib/active_record/transactions.rb

```diff
@@ -99,6 +99,16 @@ module Transactions
     # be propagated (after triggering the ROLLBACK), so you should be ready to
     # catch those in your application code.
     #
+    # Exceptions thrown by the ROLLBACK have a #message method that contains
+    # the ActiveRecord error message. 
+    #
```

Comment:
hey @raq929 thanks for the PR,
I wonder why do you mean by "Exceptions thrown by the ROLLBACK" , as far as I understand, for the example given, this error will come from somewhere else, in this case a validation error. That will eventually call a rollback, but the exception is not thrown by the rollback.